net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/general.xml

```diff
@@ -16,7 +16,7 @@
         <label>Enable VLAN fallback assignment</label>
         <type>checkbox</type>
         <advanced>true</advanced>
-        <help>This allows you to define a fallback VLAN-Group-ID. Beware setting this option will send an accepted radius reply even the authentication fails.</help>
+        <help>This allows you to define a fallback VLAN-Group-ID. Be ware setting this option will send an accepted radius reply even the authentication fails.</help>
```

Comment:
maybe "Be aware that setting this option will send an accepted RADIUS reply even if the authentication fails."

there's still a typo with "even the authentication"

Reply:
also, `RADIUS` should be capitalized

Reply:
perhaps, 
>Warning: Setting this option will send an accepted RADIUS reply even if the authentication attempt fails

Reply:
thank you. changed.